docs/asciidoc/more_background_on_compact_identifiers_content.adoc

```diff
@@ -0,0 +1,145 @@
+== About
```

Comment:
Format question -- did you think about just putting this into a .md file instead, since it doesn't need to be tightly coupled to the generated docs? I expect it will be a lot easier to maintain, since it avoids the whole build process. (And if I understand James' work, we'll be switching everything from .adoc to .md anyway..)

Reply:
Happy to revise when the new build system is in place.

Reply:
Okay -- you can use .md now and it will be future-proof (and will keep the builds simpler in the meantime), but agree it's not at all pressing.

Reply:
Looks like you want to leave it as .adoc for now, which is fine; resolving.